src/panels/lovelace/cards/hui-history-graph-card.ts

```diff
@@ -151,10 +151,11 @@ export class HuiHistoryGraphCard extends LitElement implements LovelaceCard {
   }
 
   protected shouldUpdate(changedProps: PropertyValues): boolean {
-    if (changedProps.has("_stateHistory")) {
-      return true;
-    }
-    return hasConfigOrEntitiesChanged(this, changedProps);
+    return (
+      hasConfigOrEntitiesChanged(this, changedProps) ||
+      changedProps.size > 1 ||
+      !changedProps.has("hass")
```

Comment:
would this be better wrapped in a helper function, maybe `hasNonHassChanged`, or `hasConfigOrEntitiesOrOtherChanged`? 

Don't love either name but it may be clearer than repeating this pattern everywhere.

Reply:
Yes, would be nice